custom_components/ha_washdata/profile_store.py

```
@@ -3895,12 +3929,163 @@ def to_points(y_vals: list[float]) -> list[list[float]]:
+          bounded by the ambiguity gate + cold-start floor. Grouping variants is
+          the recommended workflow and restores full coherence.
+        """
+        try:
+            for grp in self.get_profile_groups().values():
+                members = grp.get("members") if isinstance(grp, dict) else None
+                if isinstance(members, list) and program in members:
+                    sib = {m for m in members if isinstance(m, str)}
+                    if len(sib) >= 2:
+                        return sib
+        except Exception:  # noqa: BLE001
+            pass
```

Ruff (0.15.21)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)